knowledge-base/rokroskar/workflow-run-cleanup-action/action-security.yml

```diff
@@ -0,0 +1,9 @@
+name:  Workflow run cleanup action
+github-token:
+  environment-variable-name: GITHUB_TOKEN
+  permissions:
+    actions: read
```

Comment:
@h0x0er since it is cancelling action workflows, should it not be `actions: write`?